Formula/vineyard.rb

```diff
@@ -0,0 +1,102 @@
+class Vineyard < Formula
+  include Language::Python::Virtualenv
+
+  desc "In-memory immutable data manager. (Project under CNCF)"
+  homepage "https://v6d.io"
+  url "https://github.com/v6d-io/v6d/releases/download/v0.6.0/v6d-0.6.0.tar.gz"
+  sha256 "26a024bd143a19f1ba9fb87fa9eb9fec53f19836d8214b35bc472fa2132033c3"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "apache-arrow"
+  depends_on "boost"
+  depends_on "etcd"
+  depends_on "etcd-cpp-apiv3"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "libgrape-lite"
+  depends_on "llvm"
+  depends_on "open-mpi"
```

Comment:
Looks like we need `nlohmann-json` here too.

Reply:
Added.

Reply:
And also `pybind11`?

Reply:
Pybind11 is a build-time dependency and we use the bundled version in the git tarball.

The bundled pybind11's headers won't be installed.